.github/workflows/contracts.yml

```diff
@@ -189,3 +189,62 @@ jobs:
       #     token: ${{ secrets.NPM_TOKEN }}
       #     access: public
       #     dry-run: true # TODO: to be removed once testing of workflow is done
+
+  migrate-and-publish-celo:
+    needs: [build-and-test, lint]
+    if: |
+      github.ref == 'refs/heads/master'
+        && (github.event_name == 'push'
+        || github.event_name == 'workflow_dispatch')
+    environment: keep-test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    defaults:
+      run:
+        working-directory: ./solidity
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-solidity-node-modules
+        with:
+          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Migrate contracts
+        env:
+          CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY: |
+            ${{ secrets.KEEP_TEST_CELO_CONTRACT_OWNER_PRIVATE_KEY }}
+        run: npx truffle migrate --reset --network alfajores
```

Comment:
We should probably pull `alfajores` to an env property, but I have no good idea how to handle it best together with ethereum, so the current code is fine with me.

Reply:
Yeah definitely, I can leave a TODO about that.

Reply:
I pulled `alfajores` to a secret `KEEP_TEST_CELO_TRUFFLE_NETWORK`. It's consistent with how we handle it for ethereum.

Reply:
See https://github.com/keep-network/keep-core/commit/946d20a56bb31b93d968b6cfb2f9103d35dad665